timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -35,11 +35,21 @@ def run_sigma_rule(self, query, rule_name, tag_list = None):
         """
         return_fields = []
         tagged_events_counter = 0
+        # TODO: tune the below line to be more effective
         events = self.event_stream(
             query_string=query, return_fields=return_fields)
         for event in events:
-            event.add_tags(['sigma_{0:s}'.format(rule_name)])
+            ts_sigma_rule = event.source.get('ts_sigma_rule',[])
+            ts_sigma_rule.append(rule_name)
+            event.add_attributes({'ts_sigma_rule': list(set(ts_sigma_rule))})
+            ts_ttp = event.source.get('ts_ttp',[])
+            for tag in tag_list:
+                if tag.startswith(('attack.','car.')):
```

Comment:
nit, space after comma

Reply:
can you add a comment to explain why these two? why attack. and car.?

Reply:
nit is gone. attack. and car. are the ones that are currently present in Sigma rules that IMHO qualify as TTPs.